client/components/account-balances/index.tsx

```diff
@@ -130,129 +131,135 @@ const AccountBalances: React.FC = () => {
+			{ ( tab: BalanceTabProps ) => {
+				const totalBalance = tab.availableFunds + tab.pendingFunds;
+
+				const negativeBalanceTooltipContent = interpolateComponents( {
+					mixedString: __(
+						'{{learnMoreLink}}Learn more{{/learnMoreLink}} about why your account balance may be negative.',
+						'woocommerce-payments'
+					),
+					components: {
+						learnMoreLink: (
+							// eslint-disable-next-line jsx-a11y/anchor-has-content
+							<a
+								rel="external noopener noreferrer"
+								target="_blank"
+								href={ documentationUrls.negativeBalance }
+							/>
+						),
+					},
+				} );
+				return (
+					<>
 						<Flex
 							gap={ 0 }
-							className="wcpay-account-balances__instant-deposit"
+							className="wcpay-account-balances__balances"
 						>
-							<InstantDepositButton
-								instantBalance={ tab.instantBalance }
+							<BalanceBlock
+								id={ `wcpay-account-balances-${ tab.currencyCode }-total` }
+								title={ fundLabelStrings.total }
+								amount={ totalBalance }
+								currencyCode={ tab.currencyCode }
+								tooltip={
+									<BalanceTooltip
+										label={ `${ fundLabelStrings.total } tooltip` }
+										maxWidth={
+											totalBalance < 0
+												? undefined
+												: '335px'
+										}
+										content={
+											totalBalance < 0 ? (
+												negativeBalanceTooltipContent
+											) : (
+												<>
+													<>
+														{ interpolateComponents(
+															{
+																mixedString: __(
+																	'{{bold}}Total balance{{/bold}} combines both pending funds (transactions under processing) and available funds (ready for deposit). {{learnMoreLink}}Learn more{{/learnMoreLink}}',
+																	'woocommerce-payments'
+																),
+																components: {
+																	bold: <b />,
+																	learnMoreLink: (
+																		// eslint-disable-next-line jsx-a11y/anchor-has-content
+																		<a
+																			rel="external noopener noreferrer"
+																			target="_blank"
+																			href={
+																				documentationUrls.depositSchedule
+																			}
+																		/>
+																	),
+																},
+															}
+														) }
+													</>
+													<InlineNotice
+														className="wcpay-account-balances__balances-total-balance-tooltip"
+														isDismissible={ false }
+													>
+														{ __(
+															'Total balance = Available funds + Pending funds',
+															'woocommerce-payments'
+														) }
+													</InlineNotice>
+												</>
+											)
+										}
+									/>
+								}
+							/>
+							<BalanceBlock
+								id={ `wcpay-account-balances-${ tab.currencyCode }-available` }
+								title={ fundLabelStrings.available }
+								amount={ tab.availableFunds }
+								currencyCode={ tab.currencyCode }
+								tooltip={
+									<BalanceTooltip
```

Comment:
Might be worth factoring out these into semantic tooltip components, e.g. `TotalBalanceTooltip` and `AvailableBalanceTooltip`. Then if there is some logic to fine-tune the content (e.g. add info about implications of a negative total balance), it's abstracted away from this top-level component.

Reply:
Side note – `BalanceTooltip` might not be adding much value, it's effectively a synonym / wrapper for `ClickToolTip`.

Reply:
> Might be worth factoring out these into semantic tooltip components, e.g. TotalBalanceTooltip and AvailableBalanceTooltip.

Good idea. Will implement once we agree on https://github.com/Automattic/woocommerce-payments/pull/8140/files#r1498209420.